### PR TITLE
Development mycelium focus

### PR DESCRIPTION
**Pull request by Mik-TF**
# Related Issue

- #619 

# Work Done

- Put mycelium at forefront of manual
- Updated Dashboard section to put mycelium network
- Updated Get Started section with Mycelium info

# Status

- Done

# Future

- Once the Mycelium OSX + Windows client is done, we can document it in Mycelium section and all the links will work (Get started section leads to generic mycelium TOC)
- For now the changes are OK to put Mycelium at the forefront of the manual

### CODE REVIEW

**Comment by khaledyoussef24** [comment]
shall we wait untill mycelium update reaches mainnet or can we merge now ?

**Comment by Mik-TF** [comment]
We can merge to dev branch (manual.dev.grid.tf) and we will wait until it's on main to merge the changes to manual.grid.tf

**Review by khaledyoussef24 — APPROVED** [review]

**Pull request merged**
